oidc/issuance/metadata.go

```diff
@@ -0,0 +1,167 @@
+package issuance
+
+import (
+	"github.com/TBD54566975/ssi-sdk/util"
+	"github.com/goccy/go-json"
+	"github.com/pkg/errors"
+	"golang.org/x/text/language"
+)
+
+type CryptographicBindingMethodSupported string
+
+const (
+	JWKFormat CryptographicBindingMethodSupported = "jwk"
+	COSEKey                                       = "cose_key"
+	AllDIDs                                       = "did"
+)
+
+type Logo struct {
+	Url     *util.URL `json:"url,omitempty"`
+	AltText *string   `json:"alt_text,omitempty"`
+}
+
+type CredentialDisplay struct {
+	displayJSON
+
+	Logo            *Logo   `json:"logo,omitempty"`
+	Description     *string `json:"description,omitempty"`
+	BackgroundColor *string `json:"background_color,omitempty"`
+	TextColor       *string `json:"text_color,omitempty"`
+}
+
+type Format string
+
+const (
+	JwtVcJson   Format = "jwt_vc_json"
+	JwtVcJsonLd        = "jwt_vc_json-ld"
+	LdpVc              = "ldp_vc"
+)
+
+type CredentialSupported struct {
+	Format Format `json:"format" validate:"required"`
+
+	Id *string `json:"id,omitempty"`
```

Comment:
```suggestion
	ID *string `json:"id,omitempty"`
```
 
is a string ptr needed over a string?